R/Report_Setup.R

```diff
@@ -44,13 +44,10 @@ Report_Setup <- function(dfGroups = NULL, dfMetrics = NULL, dfResults = NULL) {
     pivot_wider(names_from = "Param", values_from = "Value") %>%
     as.list()
 
-  if ("protocol_number" %in% names(output$lStudy)) {
-    output$StudyID <- output$lStudy$protocol_number
-  } else if ("protocol_title" %in% names(output$lStudy)) {
-    output$StudyID <- output$lStudy$protocol_title
-  } else {
-    output$StudyID <- "Unknown"
-  }
+  output$StudyID <- dfGroups %>%
```

Comment:
@samussiah here is the implementation for #1877 , everything else has to do with the updates with the reporting data and its implications across our examples, tests, etc.

Reply:
Looks great @zdz2101!